wasm/src/main/scala/ir2wasm/WasmBuilder.scala

```diff
@@ -987,6 +988,19 @@ class WasmBuilder {
     ctx.addExport(exprt)
   }
 
+  private def genDelayedTopLevelExport(exportedName: String)(implicit ctx: WasmContext): Unit = {
```

Comment:
```suggestion
  /** Generates a delayed top-level export, that is a global variable that is initially set to a
    * null reference and be later assigned a value in start function.
    */
  private def genDelayedTopLevelExport(exportedName: String)(implicit ctx: WasmContext): Unit = {
```

Can you add a little bit of explanation?